azenta_driver/azenta_driver/sealer_client.py

```diff
@@ -36,6 +40,11 @@ def connect_sealer(self):
         return ser
 
     def send_command(self, command, success_msg ="", err_msg = ""):
+        '''
+        Sends the given command to the sealer.
+        Resets sealer before sending given command. 
+        '''
+
         ser = self.connect_peeler()
         ser.write(command.encode('*00SR=zz!'))

```

Comment:
Is it a problem that you don't pass the command string through? 

I was feeling curious/nosy about what a ROS driver looks like. So, pardon if this comment is ignorant